eclcli/vnf/v1/virtual_network_appliance_plan.py

```diff
@@ -2,10 +2,8 @@
 
 import json
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    import json
except ImportError:
    import simplejson as json
```